resallocserver/manager.py

```diff
@@ -402,6 +445,9 @@ def loop(self, event):
 
         self._allocate_more_resources(event)
 
+        # Delete all resources that are not recognized by resalloc
+        self._clean_unknown_resources(event)
+
```

Comment:
We probably don't want to run this so frequently.  The orphaned instance is a rare situation...  And the _clean_unknown_resources can be pretty expensive and may take minutes, this could lead to multiple workers in parallel I think...
Could we somewhat "lock" using the DB .... and start for example no sooner than after e.g. 30 minutes since the last run?

Reply:
> Could we somewhat "lock" using the DB .... and start for example no sooner than after e.g. 30 minutes since the last run?

I am proposing a different solution, PTAL.